app/views/issues/_table.html.erb

```diff
@@ -21,7 +21,7 @@
     </thead>
     <tbody>
       <% issues.each do |issue| %>
-        <% cache [issue, @all_columns, issue.try(:affected_count), issue.try(:state), 'issues-table', @tags] do %>
+        <% cache [issue, @all_columns, issue.try(:affected), issue.try(:state), 'issues-table', @tags] do %>
```

Comment:
Instead of using all the node details in the cache key, can we instead use the `updated_at`?

```suggestion
        <% cache [issue, @all_columns, issue.try(:affected_count), issue.affected.pluck(:updated_at), issue.try(:state), 'issues-table', @tags] do %>
```